resources/assets/coffee/react/_components/changelog-entry.coffee

```diff
@@ -24,9 +24,12 @@ export ChangelogEntry = ({entry}) =>
           a
             href: entry.url
             className: 'changelog-entry__link'
-            entry.title
+            dangerouslySetInnerHTML:
```

Comment:
this is all too much code for what's essentially just
```
_.escape(entry.title).replace(/`([^`]+)`/g, '<code>$1</code>')
```
as this is apparently what github is doing according to some basic testing (and it doesn't match commonmark spec either)

maybe if they updated it to allow proper markdown inline code then we'll adjust accordingly

Reply:
alright, I'll just keep it to this component then
I noticed it can do balanced backticks as well, but it never allows backticks inside so it's still a simple regex almost identical to what you wrote.